Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -512,6 +509,23 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
         updateContentIfNeeded()
     }
 
+    /// Updates the `textView` according to the content
+    /// - Parameter content: The content of the message
+    open func updateMessageText(for content: _ChatMessage<ExtraData>?) {
+        var textColor = appearance.colorPalette.text
+        var textFont = appearance.fonts.body
+        
+        if content?.isDeleted == true {
+            textColor = appearance.colorPalette.textDisabled
+        } else if content?.shouldRenderAsJumbomoji == true {
+            textFont = appearance.fonts.emoji
+        }
+        
+        textView?.textColor = textColor
+        textView?.font = textFont
+        textView?.text = content?.textContent
```

Comment:
nit: we don't extract setting a content/style for individual subview into a func. It absolutely makes sense to me but ATM we keep everything in `updateContent`

Reply:
Got it! Moved back the text content styling to the original method.